### DISCUSSION TITLE
Sort actions in marketplace by popularity and projects using them.

### DISCUSSION THREAD

**Discussion by shivammathur** [community/community#24934] [Actions]
<p>When searching for actions in marketplace it is difficult to find the one everyone is using. Please sort them by stars, number of projects using the action or a combination of both. Also adding usedby feature to the actions like on libraries would be a great addition.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams. Thanks again for reaching out :grinning:</p>

**Comment by keyzj** [comment]
<p>Hello!<br>
Any updates on it? Seems very reasonable feature.</p>

**Comment by f3oall** [comment]
+1
